tests/x11/evolution/evolution_prepare_servers.pm

```diff
@@ -16,12 +16,16 @@ use base "opensusebasetest";
 use testapi;
 use utils;
 use version_utils qw(is_sle is_jeos is_opensuse);
+use registration 'add_suseconnect_product';
 
 sub run() {
     select_console('root-console');
     pkcon_quit;
 
     if (check_var('SLE_PRODUCT', 'sled') || get_var('DOVECOT_REPO')) {
+        if (is_sle("15-sp1+")) {
+            add_suseconnect_product('sle-module-server-applications');
```

Comment:
From QAM perspective, I am not sure if it is a good idea to add products during test. Additionally, you definitely want to remove it afterwards.